package.json

```diff
@@ -101,6 +101,7 @@
     "redis": ">=3.1.2",
     "hosted-git-info": ">=3.0.8",
     "lodash": ">=4.17.21",
-    "ws": ">=7.4.6"
+    "ws": ">=7.4.6",
+    "glob-parent": ">=5.1.2"
```

Comment:
I don't see this package being used anywhere - was it added by accident?

Reply:
ahh it's just a resolution to address something reported by yarn audit - correct?

Reply:
yes